string name
string bond_id
uint32 state_machine_id
